rust-mode.el

```diff
@@ -616,6 +616,12 @@ the desired identifiers), but does not match type annotations \"foo::<\"."
      ;; Field names like `foo:`, highlight excluding the :
      (,(concat (rust-re-grab rust-re-ident) ":[^:]") 1 font-lock-variable-name-face)
 
+     ;; Type-inferred constant
+     (,(concat "\\_<\\(?:let\\|ref\\)\\s-+" (rust-re-grab rust-re-ident) "\\_>") 1 font-lock-constant-face)
```

Comment:
I do not think of this as a constant. It is a variable, just one that doesn't permit mutable borrows (note that interior mutation can still occur, e.g. with a `Cell`). Can you change to `font-lock-variable-name-face` and group with the regex below?